fixes the TESTERPRESENT_MESSAGE in the KW2C3PE protocol #69

rikogercke opened the pull request:
Fixes the TESTERPRESENT_MESSAGE in the KW2C3PE protocol #69 by changing it from `3E 01` to `3E 02`.

rikogercke committed:
fix the TESTERPRESENT_MESSAGE

N0cynym commented on Caesar/CaesarConnection/Protocol/KW2C3PE.cs:
```diff
@@ -153,7 +153,7 @@ public override Dictionary<string, decimal> GetDefaultComParamValues()
                 //{ CP.SWSUPPLIERBLOCK, 1 },
                 //{ CP.SWVERSIONBLOCK, 1 },
 
-                { CP.TESTERPRESENT_MESSAGE, 0x3E01 },
```
From KWP2000 specification the `0x01` - `Response Required` is default value!
I would not like to change this to `0x02` - `No Response Required`!

N0cynym reviewed (changes requested):
This should not been changed.

jglim committed:
Add quirk detection for KI211

Additional ComParams can now be added to specific ECUs via quirks.

Add quirk for KI211 to use TESTERPRESENT_MESSAGE 0x3E02

jglim committed:
Restore KW2C3PE TESTERPRESENT_MESSAGE

TESTERPRESENT_MESSAGE should default to 3E01. To change this insert a quirk into ComParamQuirks.cs
